proofs.md

```diff
@@ -1,44 +1,55 @@
+- __*piece:*__ a block of data of at most `SECTOR_SIZE` bytes which is generally is a client's file or part of.
+- __*original data:*__ the concatenation of a __*sector's*__ constitutent pieces, all __*piece padding*__, and any __*terminal padding*__.
+- __*unsealed sector:*__ a concrete representation (on disk or in memory) of a sector's __*original data*__. 
+- __*sealed sector:*__ a concrete representation (on disk or in memory) of the unique replica generated by `Seal` from an __*unsealed sector*__. 
+- __*piece padding:*__ a block of zero or more 'zero bytes' inserted between __*pieces*__ to ensure they are positioned within the containing __*sector*__ in a way compatible with the __*Piece Inclusion Proof*__.
+- __*terminal padding:*__ a block of zero or more 'zero bytes' inserted after a __*sector's*__ final piece, ensuring that the length of the __*original data*__ is `SECTOR_BYTES`.
+- __*preprocessing:*__ a transformation applied to an __*unsealed sector*__ as the first stage of sealing and which may increase the size of the data.
+- __*preprocessed data:*__ the result of __*preprocessing*__ the __*original data*__.
+- __*preprocessed sector:*__ a concrete representation (on disk or in memory) of a __*sector's*__ __*preprocessed data*__.
+- __*SNARK proof:*__ a block of bytes which proves that the creator is in possession of a satisfying assignment to a quadratic arithmetic program (__*circuit*__).
+- __*Multi-SNARK proof:*__ a block of one or more __*SNARK proofs*__, each proving partition of the total set of challenges to be proved.  
+- __*Merkle inclusion proof:*__ a proof (whose format is unspecified here) that a given leaf block is contained within a merkle tree whose root is the __*commitment*__ associated with the proof.
+- __*commitment:*__ an opaque block of data to which a prover 'commits', enabling subsequent proofs which cannot be validly constructed unless the __*commitment*__ itself was validly constructed. For example: the output of a suitable pseudorandom collision-resistant hash function may serve as a __*commitment*__ to the data which is the preimage of that hash. Publication of the __*commitment*__ proves that the creator was in possession of the preimage at the time the __*commitment*__ was generated.
+- __*prover:*__ the party who generates a proof, in Filecoin it's always the Storage Miner.
+- __*verifier:*__ the party who verifies a proof generated by a __*prover*__, in Filecoin it's a full node.
+- __*sectors count:*__ the number of sectors over which a proof-of-spacetime is performed (`POST_SECTORS_COUNT`).
+
+## Proof-of-Replication Algorithms
 
 __*Filecoin Proof of Replication*__ generates a unique copy (__*sealed sector*__) of a __*sector's*__ __*original data*__, a __*Multi-SNARK proof*__, and a set of __*commitments*__ identifying the __*sealed sector*__ and linking it to the corresponding __*unsealed sector*__.
 
-## Seal
+### PoRep.Seal
```

Comment:
I'd remove "PoRep" in the title since (1) we are already in the proof of replication section and (2) PoRep is not defined before.

It'd probably be easier to read `Seal` as a title and just before having a sentence that says that PoRep means Proof of Replication.

Reply:
ack